src/lib/process.pl

```diff
@@ -31,9 +31,9 @@
 must_be_known_options(Valid, Found, [X|XS]) :-
     X =.. [Option|_],
     (
-        member(Option, Found) -> throw(error(duplicate_option, process_create/3, Option)) ;
+        member(Option, Found) -> error(evaluation_error(duplicate_options), process_create/3);
```

Comment:
`evaluation_error` is for arithmetic expressions:

<pre>
  g) There shall be a Evaluation Error when the operands
  (3.121) of an evaluable functor are such that the
  operation has an exceptional value (9.1.2). It has the
  form evaluation_error(Error) where
</pre>

Reply:
Then my next attempt at getting the error correct would be to use 
`domain_error` as a valid value of the domain `process_create_options_list` doesn't contain duplicate/conflicting options?.

Reply:
For comparison:

<pre>
<b>?- labeling([ff,ff], []).</b>
   error(domain_error(nonrepeating_labeling_options,[ff,ff]),unknown([ff,ff])-1).
</pre>